chapters/5-processing.tex

```diff
@@ -596,8 +612,14 @@ \subsection{De-identification in practice}
 a social network module, drop all names.
 If confidential information is strictly required for the analysis and cannot be
 masked or encoded,
-it will be necessary to keep at least a subset of the data encrypted through
-the data analysis process.
+additional precautions need to be taken
+unless everyone accessing it is always working in a secure environment,
+which is rarely the case.
+In most cases,
+it will be necessary to keep at least a subset of the data 
+stored in an encrypted format through the data analysis process.
+This data will need to be decrypted and loaded to a \textit{secure environment}
+every time you run code to analyzed or process it.
```

Comment:
```suggestion
then at least the confidential part of the data is required 
to remain encrypted 
and only be decrypted when used during in the data analysis process.
It is never an excuse to store confidential data unencrypted
or saved in any other insecure way,
just because it is needed in the analysis process.
```
I think we are confusing things by bringing in _secure environment_. What we really are saying is that it is not an excuse to save confidential data insecurely just because your analysis scripts reads those files. Other parts of the book covers the details of whats secure or not. Does this sound good? I think this would have prevented Jim's confusion.

Reply:
```suggestion
at least the confidential part of the data is required 
to remain encrypted 
and only be decrypted when used during in the data analysis process.
Using confidential data in the analysis process
does *not* justify storing or sharing it in an insecure way.
```

Reply:
Does `*not*` formatting work in LaTeX? or is this markdown formatting baked in?

Reply:
I think the "then" on line 613 should still be there, but a professional reviewer is the person best suited to decide what is best. Was it intentional that you removed it? Apart from that I think your last version is ready to be committed.

Reply:
I added it back. I think it's optional, and I removed it just for stylistic reasons, but I don't feel strongly about it